python/README.md

```diff
@@ -27,11 +27,15 @@ upgrade the version in pyproject.toml
 you can install package locally before uploading it to the pypi
 
 ```shell
-    pip install dist/bxsolana-trader-proto-0.0.29.tar.gz
+    pip install dist/bxsolana-trader-proto-0.0.31.tar.gz
```

Comment:
should we just say something like `pip install dist/bxsolana-trader-proto-<your version>.tar.gz` and add a comment saying the version can be taken from pyproject.toml ? so we don't need to update this document every time we bump the version

Reply:
done, i already have a comment `upgrade the version in pyproject.toml` so, i think that's enough.